src/tokens/colors/wipro.json

```diff
@@ -2,60 +2,60 @@
   "color": {
     "wipro": {
       "cyan": {
-        "100":  { "value": "#0089cf" },
-        "80":   { "value": "#1099d6" },
-        "60" :  { "value": "#5bacdf" },
-        "40":   { "value": "#93c2e9" },
-        "20":   { "value": "#c7ddf3" },
-        "7":    { "value": "#e7f2fb" }
+        "base":     { "value": "#0089cf" },
+        "tint-80":  { "value": "#1099d6" },
```

Comment:
you've used `tint-` in this file and `shade-` in the extended file. is that because the process of generating the expanded colour range used tints and/or shades or could the naming scheme be made consistent?

Reply:
This was intentional. "Tint' and "shade" are accepted terminology for colour mixing (shades are when you mix black with a base color, tints are when you mix white).

Wipro brand provides a set of tints, but no shades. Since I was adding shades, I wanted to make the distinction clearer, so now they both have prefixes.

Reply:
That's good and pretty much what I expected 👍